src/app/(views)/managed-courses/[courseId]/modules/page.tsx

```diff
@@ -1,56 +1,160 @@
 'use client'
-
-import React from 'react'
+import React, { useState } from 'react'
 import Link from 'next/link'
-import { useParams } from 'next/navigation'
+import { useParams, useRouter } from 'next/navigation'
+import useGetModules from '@/hooks/useGetModules'
 import useGetCourses from '@/hooks/useGetCourses'
+import toast from '@/utils/toast'
 import { Loading } from '@/components'
+import {
+  CButton,
+  CContainer,
+  CCol,
+  CTableHead,
+  CTable,
+  CTableBody,
+  CTableDataCell,
+  CTableHeaderCell,
+  CTableRow,
+  CDropdown,
+  CDropdownItem,
+  CDropdownMenu,
+  CDropdownToggle,
+} from '@coreui/react-pro'
+import CIcon from '@coreui/icons-react'
+import { cilPencil, cilTrash, cilPlus, cilVideo, cilImage, cilFile, cilNotes } from '@coreui/icons'
+import ConfirmDeleteModal from '@/components/ConfirmDeleteModal'
+
 export default function Modules() {
   const params = useParams()
-  const { courseId } = params as { courseId: string }
-
+  const router = useRouter()
+  const { courseId, moduleId } = params as { courseId: string; moduleId: string }
   const { courses, fetchingCourses } = useGetCourses({ courseId })
+  const { courseModules, fetchingModules } = useGetModules({ courseId, moduleId })
+  const [deletingModule, setDeletingModule] = useState(false)
+  const [visible, setVisible] = useState(false)
```

Comment:
Use a more descriptive name. What is visible? maybe const [showDeleteModa, setShowDeleteModal]?